### ISSUE TITLE
Start in background on Windows

### ISSUE THREAD

**Issue by ltoshev** [http-party/http-server#731]
I try to run the server on background without locking my console. I use gitbash on windows.

In my package.json I have defined in scripts: { "web-server": "http-server -p 1011 test/appgen/run" }

I run: npm start web-server &

The & so it runs in background, else the server blocks the further commands from executing like my cucumber tests.  Even with & on end - The server starts but it blocks the console and I cannot start my cucmber tests.

Any idea how this can be achieved? Basically to have an option where it runs dormant and later on I can kill the process by PID when tests finish.

Can you help me with that, is this a bug in the app or?

**Comment by thornjad** [comment]
I don't believe this is a bug in http-server, but I'll leave this open in case someone else has any help. Unfortunately I don't have much experience with Windows so I don't know any more than what you've already tried.

On the other hand, is it possible to side-step this problem and have your tests start http-server when they need it?

**Comment by Lionad-Morotar** [comment]
@ltoshev 我的解决方法是使用自定义脚本，比如 powershell 中有 Start-Job 指令可以使一个命令到后台运行。

![image](https://user-images.githubusercontent.com/26913795/136364547-2d7cc16a-c8b1-4508-ac1f-894c1428f4d7.png)

但是这样就没有日志了，这是个问题，我也不清楚应该怎么解决... 如果你找到了更好的方法请告诉我，谢谢。
